src/node/media_element.rs

```diff
@@ -9,10 +9,14 @@ use super::{
     ChannelConfigOptions, MediaStreamRenderer,
 };
 
-/// Options for constructing a MediaElementAudioSourceNode
+/// Options for constructing a [`MediaElementAudioSourceNode`]
+// dictionary MediaElementAudioSourceOptions {
+//   required HTMLMediaElement mediaElement;
+// };
+// note that `MediaElementAudioSourceOptions` does not extend `AudioNodeOptions`
```

Comment:
Very sharp! Let's drop it (and fill with defaults/panics in the `AudioNode`'s required methods

Reply:
Same goes for ConstantSourceOptions